backend/coins/eth/account.go

```diff
@@ -222,6 +224,23 @@ func (account *Account) update() error {
 		return err
 	}
 
+	// Nonce to be used for the next tx, fetched from the ETH node. It might be out of date due to
+	// latency, which is addressed below by using the locally stored nonce.
+	nodeNonce, err := account.coin.client.PendingNonceAt(context.TODO(), account.address.Address)
+	if err != nil {
+		return err
+	}
+	account.nextNonce = nodeNonce
+
+	// Usually, our locally stored last nonce is used to compute the next noce. Edge case: if a user
```

Comment:
Why `usually`? Unless the user makes several transactions right after one another, `localNonce > account.nextNonce` evaluates to false and thus usually not the locally stored nonce is used.